test/integration/imagechange_buildtrigger_test.go

```diff
@@ -465,6 +465,8 @@ func TestMultipleImageChangeBuildTriggers(t *testing.T) {
 	}
 	defer watch2.Stop()
 
+	ignoreBuilds := make(map[string]string)
+
 	for _, tc := range triggersToTest {
 		imageStream := mockImageStream(tc.name, tc.tag)
 		imageStreamMapping := mockStreamMapping(tc.name, tc.tag)
```

Comment:
yeah i think it's probably poor naming.  it might be a mock in the sense that perhaps it's not as fully fleshed out as it should be or in some other way not quite valid?  Not sure.  

But you're right, it's not a mock in the traditional sense of proving stubbed out functionality.